learn/calling-java-code-from-ballerina.md

```diff
@@ -308,7 +308,7 @@ In this section, we explained how to use the `bindgen` tool to generate Ballerin
 
 The next sections provide more details on various aspects related to Java interoperability in Ballerina. 
 
-## The `bindgen` Tool
+## The 'bindgen' Tool
```

Comment:
Should this be normal quotes ('')? If it's Ballerina code it should be within these `` right?

Reply:
Currently, the TOC plugin doesn't support rendering back ticks(``). That’s why I added normal quotes temporarily until we fix [this](https://github.com/ballerina-platform/ballerina-dev-website/issues/1397).

Reply:
Same comment in other places

Reply:
Currently, the TOC plugin doesn't support rendering back ticks(``). That’s why I added normal quotes temporarily until we fix this.